tests/Controller/CategoryAdminControllerTest.php

```diff
@@ -277,6 +282,8 @@ protected function setUp()
                         return $categoryManager;
                     case 'sonata.classification.manager.context':
                         return $contextManager;
+                    case 'admin_code.template_registry':
+                        return $this->createMock(TemplateRegistryInterface::class);
```

Comment:
Have you tried `new TemplateRegistry` instead?

Reply:
I just saw everything is mocked and went with it, changed and works.